### DISCUSSION TITLE
Right click to open shulker in hotbar

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#566] [__Dismissed__] [answered]
Right click to open shulker box in hotbar without placing - like a backpack

**Comment by encode42** [comment]
This could probably be done with a plugin relatively easily. Another idea for your ideas list :wink:

**Comment by ApliNi** [comment, marked as answer]
Here is a plugin I found that should be what you need: 
```
More ways to open a shulkerbox:
You can right-click them while holding them or inside your inventory to open them.
```
https://www.spigotmc.org/resources/bsb-better-shulker-boxes-1-13-1-18.58837/
